code/modules/projectiles/projectile/beams.dm

```diff
@@ -60,6 +60,16 @@
 /obj/projectile/beam/weak/penetrator
 	armour_penetration = 50
 
+/obj/projectile/beam/leach
+	name = "leach shot"
+	damage = 18
```

Comment:
for 7 tc i can buy a makarov pistol, concealable in a box, with a lot of ammo, without needing to spend a lot of time and stamina on recharge, and it will do 30 damage per shot

Reply:
For a skirmish weapon, a 12 tc ebow is usually a better choice with no strings attached to the leach rifle as well. Infinite ammo with a delay between shots you can work around.

It's okay to make antags harder to actually engage constantly, since that's what they're going to be doing if they need something like this to keep skirmishing and fighting for whatever reason.

I did say this I think when the original pr to add the laser implant came up, but I feel like this is another redundant weapon which already has items filling the same space.